front-end/src/components/loginform.js

```diff
@@ -15,7 +16,20 @@ class LoginForm extends Component {
     e.preventDefault();
     console.log("The input values are", Object.values(this.state));
     // make axios call to backend login route
-
+    
+    // Backend address: https://agile-garden-80213.herokuapp.com/signin
+    axios
+    .post(`http://localhost:5000/signin`, {
+        email: this.state.email, 
+        password: this.state.password
+    })
+    .then(response => {
+      // Temp - set token here
+      console.log("State: token: " + response.data.token);
+    })
+    .catch(err => {
+      console.log(err);
+    });
     // if axios login call successful to go to..
     this.props.history.push('/navbar')
   }
```

Comment:
this should be in the `.then` block or else the user will be forwarded to a protected page regardless if login was successful or not